teammapper-frontend/src/app/core/services/map-sync/map-sync.service.ts

```diff
@@ -566,16 +614,18 @@ export class MapSyncService implements OnDestroy {
       this.updateAttachedMap();
     });
 
-    this.mmpService.on('undo').subscribe(() => {
+    this.mmpService.on('undo').subscribe((diff?: MapDiff) => {
       this.attachedNodeSubject.next(this.mmpService.selectNode());
+      // Updating the attached map is important because this persists changes after refresh - it's ok to keep this as a full update for now.
```

Comment:
I would remove this comment or at least remove the second part "its ok to..." as it does not help much the understanding